resources/yki/queries.sql

```diff
@@ -110,6 +110,143 @@ INSERT INTO exam_language (
   (SELECT id FROM organizer WHERE oid = :oid AND deleted_at IS NULL)
 );
 
+-- name: delete-quarantine!
+DELETE FROM quarantine WHERE id = :id;
+
+-- name: select-quarantines
+SELECT
+  q.id,
+  q.language_code,
+  q.end_date,
+  q.birthdate,
+  q.ssn,
+  q.first_name,
+  q.last_name,
+  q.email,
+  q.phone_number,
+  q.diary_number,
+  q.created
+FROM quarantine q;
```

Comment:
Voisi laittaa kaikkiin queryihin jonkun ORDER BY. En tiedä missä järjestyksessä possu nuo antaa jos ei ole, mutta näyttäis, että jos päivittää karenssin tietoja, niin järjestys muuttuu.